proxy/ParentSelection.cc

```diff
@@ -787,6 +788,13 @@ ParentRecord::Init(matcher_line *line_info)
   return Result::ok();
 }
 
+Result
+ParentRecord::InitYaml(std::string content)
```

Comment:
It's rare that it's a good idea to pass a `string` by value. I'm also kind of curious as to what this is doing. The configuration is loaded in to `root` and then ...? It seems to then be dropped.

Reply:
@SolidWallOfCode I'm still working on this so, it's not doing much of anything right now.  I'm working in the ControlMatcher now on a BuildTableFromYaml().  I'll fix this though.